internal/organizations/organizations.go

```diff
@@ -67,6 +67,9 @@ func Create(ctx context.Context, kubeClient *kubernetes.Cluster, gitea GiteaInte
 					kubernetes.EpinioOrgLabelKey:        kubernetes.EpinioOrgLabelValue,
 					"quarks.cloudfoundry.org/monitored": "quarks-secret",
 				},
+				Annotations: map[string]string{
```

Comment:
Do we need linkerd sidecars in user application pods too ?

Reply:
I think so. They will be talking to services created by the service-catalog. E.g. an application could be talking to another pod which is a Mariadb or other.

Reply:
Okie. Gotcha. Then we also need to inject `linkerd into `mariadb` right?

Reply:
`mariadb` pods (or any other service), are created in the same org with the application. This organization is an annotated namespace and thus any pods created there will have a sidecar.

Reply:
Gotcha.